papery/rendering.py

```diff
@@ -56,6 +56,12 @@ def __init__(self, config,
         if "assets" not in self.config:
             self.config['assets'] = []
 
+        if "themes_assets" not in self.config:
+            self.config['themes_assets'] = "assets"
```

Comment:
I recommend to fix config coerced to use underscore. As setting [`self.config`](https://github.com/withletters/papery/blob/master/papery/rendering.py#L44), please add the code like below.

```python
fixed_config = {}
    for k, v in config.items():
        fixed_config[re.sub('-', '_', k)] = v  # import 're' beforehand
self.config = fixed_config
```